src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatServiceBuilder.java

```diff
@@ -238,6 +268,19 @@ private static String validateHostname(String hostname) {
     }
 
     /**
+     * Sets a {@link Consumer} that performs additional configuration operations against
+     * the Tomcat {@link StandardServer} created by a {@link TomcatService}. This method can be invoked
+     * multiple times to add multiple {@code configurator}s.
+     */
+    public TomcatServiceBuilder configurator(Consumer<? super StandardServer> configurator) {
```

Comment:
configurator can be used to add/remove services to `TomcatService`. is it okay without any restriction?